lightning/Cargo.toml

```diff
@@ -29,6 +29,7 @@ unstable = []
 [dependencies]
 bitcoin = "0.26"
 
+hashbrown = { version = "0.9" }
```

Comment:
Current `hashbrown` is `0.11.2`, can bump to `0.11`, if MSRV `1.49` is OK

Can also feature-gate inside the prelude module, too.